shopify-theme-api/index.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+const Shopify = require('shopify-api-node');
+const core = require('@actions/core');
+const shopifyStore = process.env.SHOPIFY_STORE;
+const apiKey = process.env.SHOPIFY_API_KEY;
+const apiSecret = process.env.SHOPIFY_API_SECRET;
+const themeName = process.env.SHOPIFY_THEME_NAME;
+
+const shopify = new Shopify({
+  shopName: shopifyStore,
+  apiKey: apiKey,
+  password: apiSecret
+});
+
+async function run(action) {
+  try {
+    let themes = await shopify.theme.list()
+    themes.forEach(theme => {
+      if(action == 'get' || action == '') {
+        console.log(theme)
+      }
+      else if (theme.name == themeName) {
+        if(action == 'delete') {
+          shopify.theme.delete(theme.id).then
+          ((result) => core.debug(result),
+          (err) => console.error(err)
```

Comment:
When using `then` on the same line as the preceding promise, start the function on the same line as well. E.g., `shopify.theme.delete(theme.id).then(result => { [linebreak]`

or put `then` on a new line, indented: 

```js
shopify.theme.delete(theme.id)
  .then(result => core.debug(result)
  .catch(err => console.error(err)
```

When using promises, error handling goes in a `catch` block

Reply:
Alternatively, if you want to continue with `async/await` syntax, this could be:
```js
const result = await shopify.theme.delete(theme.id)
console.log(result)
```